README.md

```diff
@@ -18,6 +18,7 @@ Guidelines:
 |-------|------------|----------------|--------------|-----------------|--------------|-------------|
 | [YuNet](./models/face_detection_yunet)                | 160x120  | 1.45   | 6.22    | 12.18  | 4.04   | 86.69 |
 | [SFace](./models/face_recognition_sface)              | 112x112  | 8.65   | 99.20   | 24.88  | 46.25  | ---   |
+| [LPD-YuNet](./models/license_plate_detection_yunet/)  | 320x240  | ---    | 168.03  | 56.12  | 154.20\* |       |
```

Comment:
Channel wise quantized model runs slow on NPU. Is it possible to add the speed of tensor wise quantized model?

Reply:
unfortunately neural compressor does not provide any method to control that for now (see this issue https://github.com/intel/neural-compressor/issues/82). I will try [openppl-ppq](https://github.com/openppl-public/ppq) in the next patch.